README.rst

```diff
@@ -13,6 +13,16 @@ Installation
 
 
 
+Install before joulupukki-common
+
+After
+
+
+``pip install -r requirements.txt``
```

Comment:
You could use :
```
After

::

    pip install -r requirements.txt
    python setup.py develop

```

I think this is simplier with this syntax...
ref: http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html